### ISSUE TITLE
Firefox - contenteditable on h1 tag throws error

### ISSUE THREAD

**Issue by Olgagr** [yabwe/medium-editor#994]
Example is here: https://plnkr.co/edit/7tzumlKQ3XRY6ABFeSna?p=preview . When I try to activate MediumEditor on tag h1, Firefox throws very general error NS_ERROR_FAILURE from this line of code:

```
// function handleKeyup
if (node.getAttribute('data-medium-element') && node.children.length === 0) {
    this.options.ownerDocument.execCommand('formatBlock', false, 'p'); // this line throws error
}
```

Of course, obvious solution is not using h1 tag, but I wonder is this known error. It even has some sense as h1 is not the best candidate for contenteditable, because nesting some tags inside h1 can causes incorrect markup.

**Comment by nextend** [comment]
Same here, happens even for <p> tag if there is no children in this node. Is there any suggested solution?

**Comment by nmielnik** [comment]
Fixed via #1052 (continuation of #1020).  Thanks @nextend !
